component/otelcol/exporter/prometheus/internal/convert/convert_test.go

```diff
@@ -110,7 +118,27 @@ func TestConverter(t *testing.T) {
 									"count": 333,
 									"sum": 100,
 									"bucket_counts": [0, 111, 0, 222],
-									"explicit_bounds": [0.25, 0.5, 0.75, 1.0]
+									"explicit_bounds": [0.25, 0.5, 0.75, 1.0],
```

Comment:
As mentioned above - are bounds always guaranteed to be provided sorted or must the code sort them?